upi/openstack/security-groups.yaml

```diff
@@ -180,6 +180,28 @@
       protocol: '112'
       remote_ip_prefix: "{{ os_subnet_range }}"
 
+  - name: 'Create master-sg rule "master ingress HTTP (TCP)"'
+    openstack.cloud.security_group_rule:
+      security_group: "{{ os_sg_master }}"
+      protocol: tcp
+      port_range_min: 80
+      port_range_max: 80
+
+  - name: 'Create master-sg rule "master ingress HTTPS (TCP)"'
+    openstack.cloud.security_group_rule:
+      security_group: "{{ os_sg_master }}"
+      protocol: tcp
+      remote_ip_prefix: "{{ os_subnet_range }}"
```

Comment:
We should allow from anywhere, and not just the `os_subnet_range` -- similar the `Create master-sg rule "master ingress HTTP (TCP)` rule above.

Reply:
For reference, see the rule for IPI as it is currently implemented on master: https://github.com/openshift/installer/blob/2748047/data/data/openstack/masters/sg-master.tf#L519